operations/workflow.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+  
+BRANCH_PATH="$HOME/branch/"
+
+while getopts ":b :c" opt; do
+  case $opt in
+    b)
+      read -p "Enter the branch label: " branch
+      read -p "Enter your GitHub Name (First name Last name): " username
+      read -p "Enter your email to associate commits to: " email
+      git clone --single-branch --branch $branch https://github.com/bcgov/GDX-Analytics-microservice.git $branch
+      cd $branch
+      git config user.name "${username}"
+      git config user.email "${email}"
+      ;;
+    c)
+      read -r -p "Enter branch name to delete: " rmbranch
```

Comment:
@lclarko  can you check this dir exists before allowing to proceed? otherwise repeat this prompt?

Reply:
```
[microservice@ip-172-31-20-95 branch]$ rm fakedir/; echo $?
rm: cannot remove ‘fakedir/’: No such file or directory
1
[microservice@ip-172-31-20-95 branch]$
[microservice@ip-172-31-20-95 branch]$
rm: cannot remove ‘fakedir’: No such file or directory
[microservice@ip-172-31-20-95 branch]$ ./workflow.sh -c
Enter branch name to delete: fakedir
Are you sure you want to permanently delete branch folder /home/microservice/branch/fakedir? [y/N] y
Deleting /home/microservice/branch/fakedir ...
[microservice@ip-172-31-20-95 branch]$ echo $?
1
```

if it doesn't exist, either force the prompt again until user provides an existing folder, or exit (err code 1) with a similar error as rm gives before the rm -rf command runs later.

In the last command: `echo $?` I'm asking bash for the exit code of the last command. Your script gave no error message output, so I would have expected an exit code of 0. It actually exits with 1 (from the internal `rm -rf` itself returning 1 for trying to remove a `fakedir`, I think).

Reply:
Added Check for DIR existing